app/codes/transactionmanager.py

```diff
@@ -223,6 +224,19 @@ def econvalidator(self, cur=None):
         # from mempool only include transactions that reduce balance and not those that increase
         # check if the sender has enough balance to spend
         self.validity = 0
+        
+        if 'fee' in self.transaction:
+            fee = self.transaction['fee']
+        else:
+            fee = 0
+
+        currency = self.transaction['currency']
+        if currency == NEWRL_TOKEN_CODE:
+                if fee < NEWRL_TOKEN_MULTIPLIER:
+                    return False
```

Comment:
@Prannav-k don't return False but instead set validity to 0 and msg to "Not enough fee"

Reply:
but at later point in code, it could get changed to one . Ex: line 256 @kousthubraja